example/Dockerfile

```diff
@@ -43,7 +43,7 @@ COPY runtime/bootstrap /lambda-php-runtime/
 # Install Guzzle, prepare vendor files
 RUN mkdir /lambda-php-vendor && \
     cd /lambda-php-vendor && \
-    /opt/php-7-bin/bin/php /opt/php-7-bin/bin/composer require guzzlehttp/guzzle
+    /opt/php-7-bin/bin/php /opt/php-7-bin/bin/composer require asahasrabuddhe/lambda-php-runtime-bootstrap
```

Comment:
Why not include the source in this repo too?  It would be nice if this was a full self-contained example

Reply:
I wanted to include the runtime / vendor as a library to make it more modular. The entire source is available on my own [repository](https://github.com/asahasrabuddhe/php-lambda-runtime)

Reply:
The example is really intended only as a self-contained example of using the tool.  Unless it's a well-known dependency like Guzzle, the example code should be in this repo.